app/settings/settings.tpl.jade

```diff
@@ -0,0 +1,79 @@
+        md-tab(label="Change email")
+            md-content.md-padding
+                div(layout="row")
+                    h3 Change email
+                    md-progress-linear(flex ng-show="settings.chMailStatus.submit" md-mode="indeterminate")
+                    div.error(flex="100" md-whiteframe="2" ng-model="settings.chMailStatus.message" ng-show="settings.chMailStatus.error")
+                        h4 Change email: {{settings.chMailStatus.message}}
+                    div.success(flex="100" md-whiteframe="2" ng-show="settings.chMailStatus.success")
+                        h4 Change email: Email changed successfully. Please log out to see the changes.
+                form(name="chMailForm" ng-submit="settings.changeEmail()" layout="column" layout-gt-sm="row")
+                    md-input-container.md-block(flex)
+                        label Old email
+                        input(ng-model="settings.chMailData.oldEmail" type="email" required)
+                    md-input-container.md-block(flex)
+                        label New email
+                        input(ng-model="settings.chMailData.newEmail" type="email" required)
+                    md-input-container.md-block(flex)
+                        label Password
+                        input(ng-model="settings.chMailData.password" pattern=".{5,10}" type="password" required title="Password length must be 5 to 10 characters")
+                    md-button.md-raised(type="submit" name="chMailBtn" arial-label="Change email") Change            
+        md-tab(label="Change password")
+            md-content.md-padding
+                div(layout="row")
+                    h3 Change password
+                    md-progress-linear(flex ng-show="settings.chPassStatus.submit" md-mode="indeterminate")
+                    div.error(flex="100" md-whiteframe="2" ng-model="settings.chPassStatus.message" ng-show="settings.chPassStatus.error")
+                        h4 Change password: {{settings.chPassStatus.message}}
+                    div.success(flex="100" md-whiteframe="2" ng-show="settings.chPassStatus.success")
+                        h4 Change password: Password changed successfully. Please log out to see the changes.
+                form(name="chPassForm" ng-submit="settings.changePassword()" layout="column" layout-gt-sm="row")
+                    md-input-container.md-block(flex)
+                        label Old password
+                        input(ng-model="settings.chPassData.oldPassword" pattern=".{5,10}" type="password" title="Password length must be 5 to 10 characters" required)
+                    md-input-container.md-block(flex)
+                        label New password
+                        input(ng-model="settings.chPassData.newPassword" pattern=".{5,10}" type="password" title="Password length must be 5 to 10 characters" required)
```

Comment:
I'll do this in following days. Right now if I use ng-pattern I'd have to change the way the forms validate the submit, because ng-pattern validates the input, but doesn't prevent to do the ng-submit if the pattern isn't valid. In that way I think pattern attribute does the validation in an easier way.